thorcast/forecast.py

```diff
@@ -1,28 +1,32 @@
 import requests
 
 
-def clean_forecast(forecast):
-    return forecast.replace('. ', '.\n')
-
-
-def get_forecast(lat, lng):
+def forecast_from_api(lat, lng):
```

Comment:
I feel like this is where naming things is important and not just nitpicking.
This should be named a single verb like `fetch` or `request`.
That way the use of the file would read like:
```python
import forecast

result = forecast.fetch(lat, lng)
```

Reply:
Makes sense, I'll clean up names all around.